example/package-lock.json

```diff
@@ -0,0 +1,15388 @@
+{
+  "name": "react-native-svg-draw-example",
```

Comment:
Project already have `yarn.lock` files, update them and remove `package-lock.json` files